infrastructure/modules/vpc/main.tf

```diff
@@ -0,0 +1,82 @@
+resource "aws_vpc" "vpc" {
+  cidr_block           = var.vpc_cidr
+  enable_dns_support   = var.enable_dns_support
+  enable_dns_hostnames = var.enable_dns_hostnames
+  tags = {
+    Name        = "${terraform.workspace}-vpc"
+    Environment = terraform.workspace
+  }
```

Comment:
I would put a few more tags on this like what the project is, the repo name etc. Answer has a tagging policy that would be good to look at and try and implement https://answerconsulting.jira.com/wiki/spaces/ARCH/pages/3204907011/Tagging+Policy.